src/components/sections/create-a-wallet.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  useCreateWallet as useCreateEvmWallet,
+  useSolanaWallets,
+} from "@privy-io/react-auth";
+import { useCreateWallet as useCreateWalletExtendedChains } from "@privy-io/react-auth/extended-chains";
+import Section from "../reusables/section";
+import { toast } from "react-toastify";
+
+type SupportedExtendedChains =
```

Comment:
Might be better to import this via `@privy-io/public-api` like

import type {ExtendedChainType} from '@privy-io/public-api';